lightning/src/routing/router.rs

```diff
@@ -893,7 +893,7 @@ where L::Target: Logger {
 							}
 
 							let path_penalty_msat = $next_hops_path_penalty_msat
-								.checked_add(scorer.channel_penalty_msat($chan_id.clone(), &$src_node_id, &$dest_node_id))
+								.checked_add(scorer.channel_penalty_msat($chan_id.clone(), amount_to_transfer_over_msat, *available_liquidity_msat / 1000, &$src_node_id, &$dest_node_id))
```

Comment:
Any reason to not use `msats` for channel capacity in `channel_penalty_msat`'s interface? You are multiplying by 1000 in the implementation already.

Reply:
I guess not? felt weird cause a channel's "capacity" is never msat, but I suppose we do use `htlc_maximum_msat` without issue. Will change.